lib/profile/icon_button.dart

```diff
@@ -4,12 +4,24 @@ import 'package:flutter/material.dart';
 class ProfileIconButton extends StatelessWidget {
```

Comment:
I'm not using this class anymore. I can delete it if no one else wants to recycle it 😋